manifests/mod/cluster.pp

```diff
@@ -51,15 +51,15 @@
 class apache::mod::cluster (
   String $allowed_network,
   String $balancer_name,
-  String $ip,
+  Stdlib::IP::Address $ip,
   String $version,
-  Boolean $enable_mcpm_receive          = true,
-  String $port                          = '6666',
-  Integer $keep_alive_timeout           = 60,
-  String $manager_allowed_network       = '127.0.0.1',
-  Integer $max_keep_alive_requests      = 0,
-  Boolean $server_advertise             = true,
-  Optional[String] $advertise_frequency = undef,
+  Boolean $enable_mcpm_receive                     = true,
+  Variant[Stdlib::Port, Pattern[/^[0-9]+$/]] $port = 6666,
```

Comment:
Given the next version is a major version bump, perhaps we should just go for a breaking change and only allow Stdlib::Port?

Reply:
I strongly vote for this Option.  Give me a „go“, and I will narrow down to `Stdlib::Port`.